packages/bolt-connection/src/connection-provider/connection-provider-routing.js

```
@@ -703,7 +704,8 @@ function _isFailFastError (error) {
   return [
     DATABASE_NOT_FOUND_CODE,
     INVALID_BOOKMARK_CODE,
-    INVALID_BOOKMARK_MIXTURE_CODE
+    INVALID_BOOKMARK_MIXTURE_CODE,
+    INVALID_ARGUMENT_ERROR
```

Consider also failing fast on `Neo.ClientError.Request.Invalid`.